.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
```

Comment:
This would be good to add to a future `builder-init` compatible template in `init/` for both Victory, Radium, and `builder-react-component` archetypes.